shared/vehicles.lua

```diff
@@ -9,15 +9,17 @@ QBShared.Vehicles = {
 		['model'] = 'asbo',
 		['price'] = 4000,
 		['category'] = 'compacts',
+        ['categoryLabel'] = 'Compacts',
 		['hash'] = `asbo`,
-		['shop'] = 'pdm',
+		['shop'] = {'pdm','luxury'},
```

Comment:
Any reason why you added `asbo` to the "luxury" vehicle shop?
Was this leftover from testing?

Reply:
Yeah it was for testing to make sure multi stores worked.

Reply:
I removed the "luxury"